docs/content/direct/common-setup-step-by-step.md

```diff
@@ -0,0 +1,71 @@
+# Common Setup - step-by-step
+
+The following steps create a deployment of KubeStellar Hub
+
+1. You may want to `set -e` in your shell so that any failures in the setup or usage scenarios are not lost.
+
+1. If you ran through these scenarios previously then you will need to do a bit of cleanup first. See how it is done in the cleanup script for our E2E tests (in `test/e2e/common/cleanup.sh`).
+
+1. Set environment variables to hold KubeStellar and OCM-status-addon desired versions:
+
+    ```shell
+    export KUBESTELLAR_VERSION=0.23.0-alpha.2
+    ```
+
+1. Create a Kind hosting cluster with nginx ingress controller and KubeFlex controller-manager installed:
+
+    ```shell
+    kflex init --create-kind
+    ```
+   If you are installing KubeStellar on an existing Kubernetes or OpenShift cluster, just use the command `kflex init`.
+
+1. Update the post-create-hooks in KubeFlex to install kubestellar with the desired images:
+
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/config/postcreate-hooks/kubestellar.yaml
+    kubectl apply -f https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/config/postcreate-hooks/ocm.yaml
+    ```
+
+1. Create an inventory & mailbox space of type `vcluster` running *OCM* (Open Cluster Management)
+in KubeFlex. Note that `-p ocm` runs a post-create hook on the *vcluster* control plane
+which installs OCM on it.
+This step includes the installation of the status add-on controller.
+See [here](./architecture.md#ocm-status-add-on-agent) for more details on the add-on.
+
+    ```shell
+    kflex create its1 --type vcluster -p ocm
+    ```
+
+1. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar`
+runs a post-create hook on the *k8s* control plane that starts an instance of a KubeStellar controller
+manager which connects to the `wds1` front-end and the `its1` OCM control plane back-end.
+Additionally the OCM based transport controller will be deployed.
+
+    ```shell
+    kflex create wds1 -p kubestellar
+    ```
+
+1. (optional) Check relevant deployments and statefulsets running in the hosting cluster. Expect to

```

Comment:
This step is also not specific to not using the hub chart.

Reply:
yes and more further on, that's why I suggested 2 clean readable documents for the 2 processes